### ISSUE TITLE
Copy tags for instance

### ISSUE THREAD

**Issue by cbarensfeld** [cloud-custodian/cloud-custodian#3507]
Is there a way to copy the value of a tag and create a new tag on the same instance? Our use case is that we have several AWS accounts, but we have to use a common tag scheme for the centralized AWS billing report.

Existing tag
Key: Name
Value: test1

New tag
Key: Alfa
Value: test1

Original is not removed.

Thanks,
Chad

**Comment by kapilt** [comment]
we have a couple of variants of that but nothing that addresses that directly. rename and copy-related-tag are probably the closet in form, in many ways rename is just a copy with a delete, which could be parameterized to not delete.   although afaicr, rename isn't a universal action across resources.

I guess the question would also be if you have a centralized tag schema, why not just enforce that uniformly across accounts (which is fairly common with custodian), rather than trying to piecemeal doing copying of values across many resource types in an adhoc fashion.

**Comment by cbarensfeld** [comment]
Yes that's the plan going forward, but there are many things already running that we need to get into compliance.
